integration/helpers/usercreds.go

```diff
@@ -110,12 +108,8 @@ func GenerateUserCreds(req UserCredsRequest) (*UserCreds, error) {
 	}
 	a := req.Process.GetAuthServer()
 	sshPub := ssh.MarshalAuthorizedKey(priv.SSHPublicKey())
-	ttl := time.Hour
-	if req.CertificateTTL != 0 {
-		ttl = req.CertificateTTL
-	}
 	sshCert, x509Cert, err := a.GenerateUserTestCerts(
-		sshPub, req.Username, ttl, constants.CertificateFormatStandard, req.RouteToCluster, req.SourceIP)
+		sshPub, req.Username, time.Hour, constants.CertificateFormatStandard, req.RouteToCluster, req.SourceIP)
```

Comment:
Why not pass 1h TTL from the app access test instead of hardcoding it in the helper method?

Reply:
This is just restoring it to what it was before I modified it -- IMO it's kind of tough to figure out what the "right" thing to do here is because there's some implicit wonkery behind the scenes which means this isn't really working the way it's supposed to. I would argue that we should hard code this because it literally won't work correctly otherwise -- just because it's configurable doesn't mean it will actually do what's intended.

Reply:
Gotcha, thanks for explanation.